internal/services/account_member/model.go

```diff
@@ -17,7 +17,7 @@ type AccountMemberModel struct {
 	AccountID types.String                                            `tfsdk:"account_id" path:"account_id,required"`
 	Email     types.String                                            `tfsdk:"email" json:"email,required"`
 	Status    types.String                                            `tfsdk:"status" json:"status,computed_optional"`
-	Roles     *[]types.String                                         `tfsdk:"roles" json:"roles,optional,no_refresh"`
+	Roles     types.Set                                               `tfsdk:"roles" json:"roles,computed_optional,no_refresh"`
```

Comment:
This needed to change so that we can set its value to Unknown during the planning phase (can't be done with a native go slice)